features/support/env.rb

```diff
@@ -1,21 +1,23 @@
-require 'aruba/cucumber'
+require "aruba/cucumber"
 
-PROJECT_ROOT = File.expand_path(File.join(File.dirname(__FILE__), '..', '..')).freeze
+PROJECT_ROOT = File.expand_path(File.join(File.dirname(__FILE__), "..", "..")).freeze
 
 Aruba.configure do |config|
   config.exit_timeout = Integer ENV.fetch("ARUBA_TIMEOUT") { 120 }
 end
 
-Aruba.configure do |config|
-  config.before_cmd do |cmd|
-    set_env('JRUBY_OPTS', "-X-C #{ENV['JRUBY_OPTS']}") # disable JIT since these processes are so short lived
+if RUBY_PLATFORM == "java"
```

Comment:
We actually stopped testing against jruby in https://github.com/thoughtbot/factory_bot/commit/9f1b32e07100d9a7da5439532b14cb7c349a2563#diff-354f30a63fb0907d4ad57269548329e3, so I think we can remove all of this.

Reply:
@composerinteralia I can remove this entire block, but would that cause any issues given the existing support for jruby-specific gems? https://github.com/thoughtbot/factory_bot_rails/blob/master/Gemfile#L15-L18

Reply:
These are all test-only files that aren't included in the gem (see https://github.com/thoughtbot/factory_bot_rails/blob/master/factory_bot_rails.gemspec#L13), so removing them wouldn't have any effect on people using the gem. 

But I guess it may make it harder for someone using jruby to contribute, so we can leave it.